DEVELOPER_GUIDE.md

```diff
@@ -281,6 +285,20 @@ curl -s -X POST \
 }' http://localhost:8083/connectors
 ```
 
+#### Poxy host configuration
+There are 2 ways to route request to aep endpoint through proxy server :
+1. **Using Environment Variable** : Export poxyHost and proxyPort on each kafka node, then restart kafka connect node.
+    ```
+    export KAFKA_OPTS="-Dhttps.proxyHost=127.0.0.1 -Dhttps.proxyPort=8085 -Dhttp.proxyUser=proxyUsername -Dhttp.proxyPassword=proxyPassword"
```

Comment:
Please make sure that we keep -Dhttps options consistant for all parameters
Also note that we can use _either_ http or https.